createProject.js

```diff
@@ -50,6 +50,11 @@ function doWork(params) {
         version: params.projectVersion,
         description: params.projectDescription || params.projectName,
         repository: params.repoURL,
+        globals: {
+            "__MAPSTORE_PROJECT_CONFIG__": false,
+            "__COMMITHASH__": false,
+            "__COMMIT_DATA__": false
+        },
```

Comment:
Proposal: maybe we could move all these in the in the [@mapstore/eslint-config-mapstore](https://github.com/geosolutions-it/MapStore2/blob/master/utility/eslint/index.js) and plublish a new version

Reply:
I tried to simply edit the package.json in utility/eslint/ but didn't worked,

I can try to publish a new version of this to see if it works

Reply:
pushed version 1.0.4 with globals defined, but this seems not being inherited
so I'll leave it as it is now

otherwise I can add many  // eslint-disable-next-line no-undef 
for each line a global is used, and eslint will no longer complain about it
it's only a linting problem after all

let me know what do you think

Reply:
> so I'll leave it as it is now

ok, we can keep that as you did and change this in another task